src/main.rs

```diff
@@ -93,54 +92,59 @@ fn find_files_to_delete(folder_path: PathBuf, age: &str) -> Vec<CollectedFile> {
             created_time <= days_ago
         })
         .map(|entry| CollectedFile {
-            path: entry.path(),
-            size: entry.metadata().unwrap().len(),
+            path: entry.0,
+            size: entry.1.len(),
         });
 
     return files_to_delete.chain(files_in_folders).collect();
 }
 
 fn main() {
-    let matches = App::new("Obliviate")
+    let matches = Command::new("cargo")
         .version(crate_version!())
-        .author("Paolo Marino")
-        .about("Simple utility that removes file older than a cerain amount of days.")
+        .author(crate_authors!("\n"))
+        .about("Simple utility that removes file older than a certain amount of days.")
         .arg(
-            Arg::with_name("age")
-                .short("a")
+            Arg::new("age")
+                .short('a')
                 .long("age")
+                .value_parser(value_parser!(u64))
                 .value_name("AGE")
                 .help("Number of days the file should be old to be removed.")
-                .default_value("30")
-                .takes_value(true),
+                .num_args(1)
+                .default_value("30"),
         )
         .arg(
-            Arg::with_name("dry-run")
-                .short("d")
+            Arg::new("dry-run")
+                .short('d')
                 .long("dry-run")
+                .action(ArgAction::SetTrue)
                 .help("When provided no files are deleted."),
         )
         .arg(
-            Arg::with_name("PATH")
-                .help("Path where to look for file to delete.")
+            Arg::new("PATH")
+                .help("Path where to look for files to delete.")
                 .required(true)
+                .num_args(1)
                 .index(1),
         )
         .arg(
-            Arg::with_name("verbose")
-                .short("v")
+            Arg::new("verbose")
+                .short('v')
                 .long("verbose")
+                .action(ArgAction::SetTrue)
                 .help("Outputs verbose logs to track which files are deleted."),
         )
         .get_matches();
 
-    let age = matches.value_of("age").unwrap_or("30");
-    let dry_run = matches.is_present("dry-run");
-    let verbose = matches.is_present("verbose");
+    // NOTE: This will always return Some(value) if default_value has been set.
+    let age: u64 = *matches.get_one("age").expect("Age option should always exist");
+    let dry_run = matches.get_flag("dry-run");
+    let verbose = matches.get_flag("verbose");
     let log_level = if verbose { Level::Trace } else { Level::Info };
 
-    let start_path = matches
-        .value_of("PATH")
+    let start_path: &String = matches
+        .get_one::<String>("PATH")
```

Comment:
Do I understand it correctly that this way we are allowing more than one path to be specified?

Reply:
I took some time to reply, but now I am pretty sure the initial software and my PR are not designed for repeating paths.
Do you want me to implement that ?

Reply:
No worries! 😄 

Yea, the initial software doesn't support multiple paths, but I guess it could be useful!
Feel free to implement that!

Reply:
Can this be merged as is? 😄

Reply:
Yep!